### PR TITLE
Relase note for v0.14.0

### PR DESCRIPTION
**Pull request by qiujian16**

### CODE REVIEW

**Comment by qiujian16** [comment]
/assign @mikeshng @dhaiducek 

Please check if anything needs to be added.

File: content/en/community/releases.md [hunk 1]
```diff
@@ -3,9 +3,40 @@ title: Releases
...
+- clusteradm v0.9.0 [changelog](https://github.com/open-cluster-management-io/clusteradm/releases/tag/v0.9.0)
+
+### Addons
+- config-policy-controller v0.14.0 [changelog](https://github.com/open-cluster-management-io/config-policy-controller/releases/tag/v0.14.0)
+- governance-policy-framework-addon v0.14.0 [changelog](https://github.com/open-cluster-management-io/governance-policy-framework-addon/releases/tag/v0.14.0)
+- governance-policy-propagator v0.14.0 [changelog](https://github.com/open-cluster-management-io/governance-policy-propagator/releases/tag/v0.14.0)
+- governance-policy-addon-controller v0.14.0 [changelog](https://github.com/open-cluster-management-io/governance-policy-addon-controller/releases/tag/v0.14.0)
+- multicloud-operators-subscription v0.14.0 [changelog](https://github.com/open-cluster-management-io/multicloud-operators-subscription/releases/tag/v0.14.0)
+- multicloud-operators-channel v0.14.0 [changelog](https://github.com/open-cluster-management-io/multicloud-operators-channel/releases/tag/v0.14.0)
+- multicloud-integrations v0.14.0 [changelog](https://github.com/open-cluster-management-io/multicloud-integrations/releases/tag/v0.14.0)
```
Comment (mikeshng) [inline content/en/community/releases.md:36]:
```suggestion
```

The app team is planning to archive this repo on ocm-io.

File: content/en/community/releases.md [hunk 2]
```diff
@@ -3,9 +3,40 @@ title: Releases
...
+- Exclude terminating clusters from Placementdecision: now cluster in terminating state will not be picked by the
+placement.
+- Send available condition events for managed cluster: klusterlet agent will send kubenerte event to the cluster
+namespace upon status change, which makes it eaiser for user to check the state change of the managed cluster.
+- Set install namespace of addonTemplate from addonDeploymentConfig: user of AddonTemplate API can now use 
+AddonDeploymentConfig API to set the installation namespace of the agent.
+- Configurable controller replicas and master node selector: user can not set flags `deployment-replicas` and
+`control-plane-node-label-selector` on klusterlet-operator to customize the replicas of the klusterlet agent.
+- Add capi into join flow: user can now use the join command to add a cluster-api provisioned cluster into ocm hub
+from cluster-api management cluster with `--capi-import` and  `--capi-cluster-name` flag.
```
[the remark this answers is not in the document]
Comment (dhaiducek) [reply content/en/community/releases.md:22]: Adding Policy updates:
```suggestion
- Exclude terminating clusters from Placementdecision: now cluster in terminating state will not be picked by the
placement.
- Send available condition events for managed cluster: klusterlet agent will send kubenerte event to the cluster
namespace upon status change, which makes it eaiser for user to check the state change of the managed cluster.
- Set install namespace of addonTemplate from addonDeploymentConfig: user of AddonTemplate API can now use 
AddonDeploymentConfig API to set the installation namespace of the agent.
- Configurable controller replicas and master node selector: user can not set flags `deployment-replicas` and
`control-plane-node-label-selector` on klusterlet-operator to customize the replicas of the klusterlet agent.
- Add capi into join flow: user can now use the join command to add a cluster-api provisioned cluster into ocm hub
from cluster-api management cluster with `--capi-import` and  `--capi-cluster-name` flag.
- (Policy Framework) `OperatorPolicy` enhancement and stabilization: OperatorPolicy can now be set to `mustnothave` to perform cleanup, alongside stabilization improvements.
- (Policy Framework) Add `ConfigurationPolicy` diff to the status: Adds a `recordDiff: InStatus` as the default value if it's not set. In this mode, the difference between the policy and the object on the managed is returned in the status. Sensitive data types are not displayed unless `InStatus` is explicitly provided.
- (Policy Framework) Add `recreateOption` to `ConfigurationPolicy`: When a user needs to update an object with immutable fields, the object must be replaced. In this case, `recreateOption` can be set to either `ifRequired` or `Always` depending on the requirements.
```

**Review by dhaiducek — CHANGES_REQUESTED** [review]
Sorry for the delay! I have one comment on wording and one comment on adding Policy Framework updates.

**Comment by mikeshng** [comment]
Looks good leaving it to Dale for a final look. Thanks.

**Review by dhaiducek — APPROVED** [review]
Oh no! I didn't realize this was still hanging out here. Approving.

**Pull request merged**